modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable camelcase */
+import type {GLTF} from '../../types/gltf-types';
+
+import GLTFScenegraph from '../../api/gltf-scenegraph';
+import {
+  ClassProperty,
+  EXR_feature_metadata_class_object,
+  EXT_feature_metadata_feature_table,
+  FeatureTableProperty,
+  GLTF_EXT_feature_metadata
+} from '../../types/gltf-json-schema';
+
+/** Extension name */
+const EXT_FEATURE_METADATA = 'EXT_feature_metadata';
+
+export const name = EXT_FEATURE_METADATA;
+
+export async function decode(gltfData: {json: GLTF}): Promise<void> {
+  const scenegraph = new GLTFScenegraph(gltfData);
+  decodeExtFeatureMetadata(scenegraph);
+}
+
+/** Decode one meshopt buffer view */
+function decodeExtFeatureMetadata(scenegraph: GLTFScenegraph): void {
+  const extension: GLTF_EXT_feature_metadata | null = scenegraph.getExtension(EXT_FEATURE_METADATA);
```

Comment:
Is this extension always on the root object?

Reply:
This extension is also used in primitives extensions data for [feature identification](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_feature_metadata#feature-identification) purposes.